AdClickthroughWorkerTests.swift

```diff
@@ -0,0 +1,113 @@
+//  Copyright 2019, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import XCTest
+import PlayerCore
+@testable import VerizonVideoPartnerSDK
+
+class AdClickthroughWorkerTests: XCTestCase {
+    
+    let testURL = URL(string: "http://")!
+    let mp4Creative = AdCreative.MP4(url: URL(string: "http://")!,
+                                     clickthrough: URL(string: "http://")!,
+                                     pixels: AdPixels(),
+                                     id: nil,
+                                     width: 10,
+                                     height: 10,
+                                     scalable: false,
+                                     maintainAspectRatio: true)
+    let vpaidCreative = AdCreative.VPAID(url: URL(string: "http://")!,
+                                         adParameters: "",
+                                         clickthrough: URL(string: "http://")!,
+                                         pixels: AdPixels(),
+                                         id: "")
+    let vpaidCreativeWithoutCllick = AdCreative.VPAID(url: URL(string: "http://")!,
+                                                      adParameters: "",
+                                                      clickthrough: nil,
+                                                      pixels: AdPixels(),
+                                                      id: "")
+    
+    var isShowSafariCalled: Bool!
```

Comment:
Can it be just `false`? I think we don't need `Implicitly Unwrapped Optional` here.